src/Console/Command/HelpCommand.php

```diff
@@ -451,7 +452,11 @@ function (FixerInterface $a, FixerInterface $b) {
             }
 
             if ($fixer instanceof DeprecatedFixerInterface) {
-                $description .= sprintf(' DEPRECATED: use `%s` instead.', $fixer->getSuccessor());
+                $successors = $fixer->getSuccessorsNames();
+                $message = null === $successors
+                    ? 'not supported anymore'
```

Comment:
This is not true: a deprecated fixer is still supported. The message should rather say that the fixer will be removed on next major version.